src/fragments/lib/push-notifications/ios/getting_started/10_pre_req.mdx

```diff
@@ -0,0 +1,7 @@
+import ios0 from "/src/fragments/lib/ios-prereq-category.mdx";
+
+<Fragments fragments={{ios: ios0}} />
+
+In order to use APNs, you need to setup credentials (keys or certificates) and export your credentials to a p12 file. See [Setting up push notification services](/lib/push-notifications/setup-push-service) for more information.
```

Comment:
This is the very first mention of APNs that customers will see in the docs, should we briefly say what it is, or at least what it stands for?

Reply:
good call, updated.